src/components/ProjectFilterOptionsDialog/ProjectFilterOptionsDialog.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, makeStyles } from '@material-ui/core';
+import * as React from 'react';
+import { ProjectStatus, Sensitivity } from '../../api';
+import { displayStatus } from '../../api/displayStatus';
+import { DialogForm, DialogFormProps } from '../Dialog/DialogForm';
+import { RadioField, RadioOption, SubmitError } from '../form';
+import { SwitchField } from '../form/SwitchField';
+
+const useStyles = makeStyles(({ spacing, typography }) => ({
+  checkboxGrid: {
+    display: 'flex',
+    flexFlow: 'row wrap',
+    alignItems: 'center',
+    paddingTop: spacing(2),
+  },
+  fieldLabel: {
+    fontWeight: typography.weight.bold,
+    marginBottom: spacing(2),
+  },
+}));
+
+interface FormValues {
+  status: ProjectStatus | 'ShowAll';
+  sensitivity: Sensitivity | 'ShowAll';
+  clusters: boolean;
+}
+
+export type ProjectFilterOptionsDialogProps = DialogFormProps<FormValues>;
+
+export const ProjectFilterOptionsDialog = (
+  props: ProjectFilterOptionsDialogProps
+) => {
+  const classes = useStyles();
+
+  console.log('props', props);
+
+  const initialValues: FormValues = props.initialValues
+    ? props.initialValues
+    : {
+        status: 'ShowAll',
+        sensitivity: 'ShowAll',
+        clusters: false,
+      };
+
+  return (
+    <DialogForm
+      DialogProps={{
+        fullWidth: true,
+        maxWidth: 'xs',
+      }}
+      title="Filter Options"
+      closeLabel="Reset Filters"
+      submitLabel="Apply Filters"
+      {...props}
+      initialValues={initialValues}
+    >
+      <RadioField name="status" label="Status">
```

Comment:
It's not clear from designs but we actually want these to be checkboxes instead. `CheckboxesField`

Reply:
Checkboxes means multiple options can be selected right? Is there a way to make it so that all but one are selected in the event that `ShowAll` is selected?